libmamba/src/core/subdirdata.cpp

```diff
@@ -480,7 +480,7 @@ namespace mamba
                 LOG_DEBUG << "Copying repodata cache files from '" << m_expired_cache_path.string()
                           << "' to '" << m_writable_pkgs_dir.string() << "'";
                 fs::u8path writable_cache_dir = create_cache_dir(m_writable_pkgs_dir);
-                auto lock = LockFile::create_lock(writable_cache_dir);
+                auto lock = LockFile(writable_cache_dir);
```

Comment:
I didn't try to change the current usage of LockFile objects because their is many cases where failing to lock is not considered an issue. So not sure.